lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -219,6 +219,15 @@ reconnect_interval:
     config_set_append: '<state> reconnect-interval <seconds>'
     default_value: 60
 
+route_distinguisher:
+  default_value: ""
+  cli_ios_xr:
+    config_get_token_append: '/^rd (\S+)$/'
+    config_set_append: ' <state> rd <rd>'
```

Comment:
For now yes. They have changed their API to be able to do a 'service cli noindent' or similar - but we need to have a discussion about how to do this in our code. We have a US to clean this all up btw.